libraries/AP_Vehicle/AP_Vehicle.cpp

```diff
@@ -95,6 +95,12 @@ void AP_Vehicle::init_vehicle()
 #endif
 }
 
+void AP_Vehicle::register_scheduler_delay_callback()
```

Comment:
Yes, it's an unrelated change (and in a separate commit, "AP_Vehicle: formatting fixes").  I moved it to make it's implementation appear in the same order as it's declaration in the .h file.  Perhaps it's sligtly OCD of me but in Copter we've done an OK job of keeping things organised so I'd like to apply some of that organisation to AP_Vehicle because it looks like more Copter code is going to move there.